# Request
float64 water
float64 iodine_level
---
# Response
bool success
string message
